### PR TITLE
kernel/binary_manager: change priority of threads and loading thread name

### PR DESCRIPTION
**Pull request by jeongarmy**
Binary manager has the highest priority, 255 and loading thread is created with priority 200.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [9e0da39a0cd752e225cd898759621e81261ec42c] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [9e0da39a0cd752e225cd898759621e81261ec42c] - Code Rule Check (C++) OK.

File: os/kernel/binary_manager/binary_manager.h [hunk 1]
```diff
@@ -33,12 +33,12 @@
...
 #define BINARY_MANAGER_NAME        "binary_manager"           /* Binary manager main thread name */
 #define BINARY_MANAGER_STACKSIZE   2048                       /* Binary manager main thread stack size */
-#define BINARY_MANAGER_PRIORITY    225                        /* Binary manager main thread priority */
+#define BINARY_MANAGER_PRIORITY    255                        /* Binary manager main thread priority */
 
 /* Loading Thread information */
-#define LOADINGTHD_NAME           "loading_thread"            /* Loading thread name */
-#define LOADINGTHD_STACKSIZE      2048                        /* Loading thread stack size */
-#define LOADINGTHD_PRIORITY       220                         /* Loading thread priority */
+#define LOADINGTHD_NAME            "loading_thread"            /* Loading thread name */
```
Comment (sunghan-chang) [inline os/kernel/binary_manager/binary_manager.h]: Let's rename this like "bm_loader"

File: os/kernel/binary_manager/binary_manager.h [hunk 2]
```diff
@@ -33,12 +33,12 @@
  ****************************************************************************/
 #define BINARY_MANAGER_NAME        "binary_manager"           /* Binary manager main thread name */
 #define BINARY_MANAGER_STACKSIZE   2048                       /* Binary manager main thread stack size */
-#define BINARY_MANAGER_PRIORITY    225                        /* Binary manager main thread priority */
+#define BINARY_MANAGER_PRIORITY    255                        /* Binary manager main thread priority */
```
Comment (sunghan-chang) [inline os/kernel/binary_manager/binary_manager.h]: Let's remove "main" from comments.

**Comment by seinfra** [comment]
Target : [f75e283dac51a596f33baa0fb29cd3b446d2ff61] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [f75e283dac51a596f33baa0fb29cd3b446d2ff61] - Code Rule Check (C++) OK.

**Pull request merged**